library.gradle

```diff
@@ -49,8 +49,6 @@ dependencies {
     testImplementation 'androidx.test:rules:1.4.0'
     testImplementation 'androidx.test.ext:junit:1.1.3'
     testImplementation 'org.robolectric:robolectric:4.6.1'
-    testImplementation 'com.squareup.okhttp3:mockwebserver:4.2.0'
-    testImplementation 'org.mockito:mockito-core:3.0.0'
```

Comment:
These are a bunch of libraries we are not using... Looks like we are not using mockito anywhere, and `mockwebserver` only in one of the modules.